### PR TITLE
Setting global defaults for styles

### LINKED ISSUE #50: Setting global defaults for styles
- Font
- Font size
- Colour palette

All this will be set in `tailwind.config.ts`. Refer these values from Figma.

### PR DESCRIPTION
**Pull request by SilentJungle399**
Fixes #50

### CODE REVIEW

File: tailwind.config.ts [hunk 1]
```diff
@@ -14,6 +14,60 @@ const config: Config = {
...
+      display: ['64px', { lineHeight: '72px' }],
+      h1: ['56px', { lineHeight: '64px' }],
+      h2: ['48px', { lineHeight: '56px' }],
+      h3: ['40px', { lineHeight: '48px' }],
+      h4: ['32px', { lineHeight: '40px' }],
+      h5: ['24px', { lineHeight: '32px' }],
+      h6: ['20px', { lineHeight: '28px' }],
+      p: ['16px', { lineHeight: '24px' }],
+      label: ['14px', { lineHeight: '22px' }],
+      tiny: ['12px', { lineHeight: '20px' }],
```
Comment (GetPsyched) [inline tailwind.config.ts:36]: https://github.com/isac-nitkkr/node-modules/issues/50#issuecomment-1917598888

Comment (SilentJungle399) [reply tailwind.config.ts:36]:
```ts
    fontSize: {
      '6xl': ['64px', { lineHeight: '72px' }],
      '5xl': ['56px', { lineHeight: '64px' }],
      '4xl': ['48px', { lineHeight: '56px' }],
      '3xl': ['40px', { lineHeight: '48px' }],
      '2xl': ['32px', { lineHeight: '40px' }],
      xl: ['24px', { lineHeight: '32px' }],
      lg: ['20px', { lineHeight: '28px' }],
      base: ['16px', { lineHeight: '24px' }],
      sm: ['14px', { lineHeight: '22px' }],
      xs: ['12px', { lineHeight: '20px' }],
    },
```

Is this fine? The labels are from [TailwindCSS docs](https://tailwindcss.com/docs/font-size)

Comment (GetPsyched) [reply tailwind.config.ts:36]: This is good. Can you make `xs`, etc. also inside quotes for consistency?

Comment (SilentJungle399) [reply tailwind.config.ts:36]: Okay

Comment (SilentJungle399) [reply tailwind.config.ts:36]: This will create warnings from prettier. Should I put it under prettier-ignore or keep as it is?

Comment (GetPsyched) [reply tailwind.config.ts:36]: Keep it as is then.

Comment (SilentJungle399) [reply tailwind.config.ts]: Pushed the changes.

**Pull request merged**